src/main/java/org/logstash/beats/BeatsHandler.java

```diff
@@ -7,23 +7,28 @@
 
 import java.io.IOException;
 import java.net.InetSocketAddress;
+import java.util.Objects;
```

Comment:
review note: Java source changes are standardization and formats. No logic changes.